src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -283,7 +283,7 @@ private function fileExcerpt(string $file, int $line, int $srcContext = 3): stri
             }
 
             for ($i = max($line - $srcContext, 1), $max = min($line + $srcContext, \count($content)); $i <= $max; ++$i) {
-                $lines[] = '<li'.($i == $line ? ' class="selected"' : '').'><a class="anchor" name="line'.$i.'"></a><code>'.$this->fixCodeMarkup($content[$i - 1]).'</code></li>';
+                $lines[] = '<li'.($i == $line ? ' class="selected"' : '').'><a class="anchor" id="line'.$i.'"></a><code>'.$this->fixCodeMarkup($content[$i - 1]).'</code></li>';
```

Comment:
generating an id using the line number alone is wrong. That's not ensuring uniqueness as multiple file excerpt might have similar line numbers in different files (or even in the same file)

Reply:
I was scared to modify the code too much.
I just now put on my big boy pants and removed them since they could not be not functional anyways (`name` tags on anchors should be unique, just like ids).

Another way of doing things would be to add the file name **and** the file "id" (`(...)/Kernel.php:line_30`, `(...)/Kernel.php-1:line_30`) but at this point we should just remove them.